.github/workflows/build-and-test.yml

```diff
@@ -117,20 +144,35 @@ jobs:
            -DCMAKE_C_COMPILER_LAUNCHER=ccache                                                          \
            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache                                                        \
            ${GITHUB_WORKSPACE}
-      - run: ccache -p
-      - run: ccache -z
-      - name: CMake Build
-        working-directory: ./build
+      - name: CMake Build (Ubuntu)
+        if: ${{ startsWith(matrix.config.os, 'ubuntu') }}
+        working-directory: ./build_${{ matrix.config.conan_profile }}
         run: cmake --build .
-      - name: CCache Stats
-        run: ccache -s
-      - name: CMake Test
-        working-directory: ./build
+      - name: CMake Test (Ubuntu)
+        if: ${{ startsWith(matrix.config.os, 'ubuntu') }}
+        working-directory: ./build_${{ matrix.config.conan_profile }}
         run: CTEST_OUTPUT_FAILURE=1 ARGS="-E IntegrationTest" cmake --build . --target test
+      - name: Install conan config (Windows)
+        if: ${{ startsWith(matrix.config.os, 'windows') }}
+        run: |
+          ./third_party/conan/configs/install.ps1
```

Comment:
For symmetry reasons, you could also just call `py -3 -m conans.conan config install .\third_party\conan\configs\windows\` here.

Reply:
Done.